src/GraphQL.ApiTests/ApiApprovalTests.PublicApi.GraphQL.approved.txt

```diff
@@ -2004,15 +2004,21 @@ namespace GraphQL.Types
     {
         public QueryArgument() { }
     }
-    public class QueryArguments : System.Collections.Generic.IEnumerable<GraphQL.Types.QueryArgument>, System.Collections.IEnumerable
+    public class QueryArguments : System.Collections.Generic.List<GraphQL.Types.QueryArgument>
     {
         public QueryArguments(params GraphQL.Types.QueryArgument[] args) { }
         public QueryArguments(System.Collections.Generic.IEnumerable<GraphQL.Types.QueryArgument> list) { }
-        public int Count { get; }
+        public new int Count { get; }
         public GraphQL.Types.QueryArgument this[int index] { get; set; }
         public void Add(GraphQL.Types.QueryArgument argument) { }
+        public void AddRange(System.Collections.Generic.IEnumerable<GraphQL.Types.QueryArgument> arguments) { }
         public GraphQL.Types.QueryArgument Find(string name) { }
-        public System.Collections.Generic.IEnumerator<GraphQL.Types.QueryArgument> GetEnumerator() { }
+        public void Insert(int index, GraphQL.Types.QueryArgument argument) { }
+        public void InsertRange(int index, System.Collections.Generic.IEnumerable<GraphQL.Types.QueryArgument> arguments) { }
```

Comment:
Why these methods? Are they needed?

Reply:
Natural consequence of inheriting from List